fusor-ember-cli/app/components/tr-subscription-manifest.js

```diff
@@ -2,4 +2,22 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   tagName: 'tr',
+
+  qtyColumn: Ember.computed('isDisconnected',
+                            'subscription.quantity_to_add',
+                            'subscription.quantity_attached',
+                            'subscription.qtySumAttached', function() {
+      if (this.get('isDisconnected')) {
+          return this.get('subscription.quantity_attached');
+
+      } else if (this.get('subscription.quantity_to_add') > 0) {
+          return this.get('subscription.quantity_attached') + ' + ' +
+                 this.get('subscription.quantity_to_add') + ' = ' +
+                 this.get('subscription.qtySumAttached');
+      } else {
+          return this.get('subscription.quantity_attached');
+      }
+
```

Comment:
I'm showing where `4 + 3 = 7` in the column if 3 were added, but only showing 4 if none were added.
It made more sense to me than adding columns. We can get UX's take